azerothcore/azerothcore-wotlk
# fix(Script/Instance): Ulduar - Yogg-Saron hardcode & some adustments.

Fixes #11859



## Changes Proposed:
- Added missing creature_text to Yogg.
- Fixed Sara going under the map
- Fixed Death Orbs (Death Ray)
- Added missing RP and texts.

## Issues Addressed:

- Closes #11859

## SOURCE:


## Tests Performed:

- tested in-game


## How to Test the Changes:


Test with 2+ characters
1. delete cache!
2. Comment out (So you can straight pull Yogg)
` if (m_pInstance->GetData(TYPE_VEZAX) != DONE || m_pInstance->GetData(TYPE_XT002) != DONE)
                return;`
3. .aura 64169 - Sanity aura
4. .go xyz 1866 -23 335 603 - tp in front of Yogg
Check the gossip, check Sara position in Phase 2, check Death Ray's movement

## Known Issues and TODO List:


- [x] fix znCN locale (allegedly)
- [x] Fix death ray being idle

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
